parachain/src/chain_spec.rs

```diff
@@ -104,7 +104,7 @@ pub fn local_config(id: ParaId) -> ChainSpec {
             serde_json::from_value(json!({
                 "ss58Format": 42,
                 "tokenDecimals": [10, 8],
-                "tokenSymbol": ["DOT", "interBTC"]
+                "tokenSymbol": ["KSM", "kBTC"]
```

Comment:
I think this kBTC renaming decision hasn't been finalized yet, right?

Reply:
Is KINT not required here?

Reply:
I think kBTC has been finalized, it was just iBTC that is still undecided (cc: @alexeiZamyatin).

Reply:
Yeah we can add KINT here too.